informal_update/signals.py

```diff
@@ -0,0 +1,40 @@
+import json
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.template.loader import render_to_string
+from django.contrib.auth.models import User
+
+from .models import InformalUpdate, InformalEmailSubscriptions
+from notifications.notification import send_notification
+from informal_update.serializers import InformalUpdateSerializer
+
+
+@receiver(post_save, sender=InformalUpdate)
+def send_email_when_informal_update_created(sender, instance, created, **kwargs):
+    if created:
+        share_with_group = instance.share_with
+        email_subscription = InformalEmailSubscriptions.objects.get(
+            share_with=str(share_with_group)
+        )
+        group = email_subscription.group
+        if group:
+            users = User.objects.filter(groups__name=email_subscription.group.name)
+            users_emails = [user.email for user in users]
+            informal_update = InformalUpdateSerializer(instance)
+            data = json.loads(json.dumps(informal_update.data))
```

Comment:
why?

Reply:
To render informal update data into into html email template

Reply:
Why not do this?
`data = informal_update.data`
instead of 
`data = json.loads(json.dumps(informal_update.data))`?